examples/experimental/onnx/post_training_quantization.py

```diff
@@ -0,0 +1,21 @@
+from examples.experimental.onnx.dataloader import create_train_dataloader
+from nncf.experimental.onnx.engine import ONNXEngine
+from nncf.experimental.post_training_api.compression_builder import CompressionBuilder
+from nncf.experimental.onnx.quantization.algorithm import ONNXPostTrainingQuantization
+from nncf.experimental.post_training_api.quantization.config import DEFAULT
+
+fp32_model_onnx_path = ''
+dataset_dir = ''
+input_shape = []
+
+# Step 1: Initialize the data loader.
+dataloader = create_train_dataloader(dataset_dir=dataset_dir, input_shape=input_shape)
+# Step 2: Initialize the engine for metric calculation and statistics collection.
+engine = ONNXEngine()
+
+# Step 3: Create a pipeline of compression algorithms.
+builder = CompressionBuilder()
+quantization = ONNXPostTrainingQuantization(DEFAULT, engine, dataloader)
```

Comment:
One of the requirements for one tool is to make optimization algorithms sharable across the supported backends (ONNX, Torch, TF).

I would make `engine` the last argument and let it be the default value.

Reply:
Great comment! It is the thing that I would like to discuss - **One of the requirements for one tool is to make optimization algorithms sharable across the supported backends (ONNX, Torch, TF).**
The current issue with that - is that implementing this needs a refusing of the inheritance of `PostTrainingQuantization ` from `BaseCompressionAlgorithmBuilder`. We should define whether we need to based on current NNCF architecture.